rfc/rfc008-certificate-structure.md

```diff
@@ -33,98 +33,47 @@ When it comes to certificates, we’d like to be at the edge of usability and se
 ## 2. Terminology
 
 * **CA**: Certificate authority as defined by [RFC2459](https://tools.ietf.org/html/rfc2459).
-* **event type**: the type of entry in the document log.
+* **CPS**: Certification Practise Statement.
+* **CRL**: Certificate Revocation List. 
 * **node**: a running instance of a Nuts implementation \(a.k.a. Nuts node\).
 * **node operator**: an organization running a node. Typically a vendor that creates healthcare software for care organizations.
-* **shared network state**: the accumulation of events or document log entries that are in the right order and where every node agrees upon.
-* **truststore**: The file that contains the trusted CA's.
+* **PKIo**: Public Key Infrastructure overheid. The by the Dutch government controlled PKI structure.
+* **TSP**: Trusted Service Provider. Party that can issue PKIo certificates. 
 
 Other terminology is taken from the [Nuts Start Architecture](rfc001-nuts-start-architecture.md#nuts-start-architecture).
 
-## 3. Overview
+## 3. PKIoverheid
 
-Nodes establish trust by using a PKI infrastructure. This infrastructure doesn't rely on central certificate authorities \(CA\) but rather on 3rd party proofs. Each vendor is a CA with a root certificate. Nodes trust a CA by either manually adding it to a truststore or relying on another cryptographic proof. The first mechanism relies on out-of-band distribution and face-to-face identity checking. For the second mechanism, all nodes should agree on which proofs to trust. This is not part of the specification, it should be part of some network agreement. [RFC006 Distributed Registry](https://github.com/nuts-foundation/nuts-specification/tree/4127385cab773917fd5bd642dd7f2fa94f48655e/rfc/rfc006-distributed-registry.md) explains which type of proofs must be supported.
+The PKIoverheid CA tree is the choice of CA to trust for production use. It's widely trusted and supported. 
+There are multiple vendors that can issue a certificate. 
+A PKIo certificate MUST be used to secure inter-node connections as well as the peer-to-peer connections used to exchange data.
```

Comment:
PKIo TLS certificate

Add specific PKIo tree/domain that applies

Reply:
Make it SHOULD instead of MUST when nodes want to communicate on their own responsibility? Or something like: Nodes MUST NOT automatically trust any other certificate than from the PKIo CA.?